src/PortingAssistant.Client.Common/PortingAssistant.Client.Common.csproj

```diff
@@ -6,7 +6,7 @@
   <ItemGroup>
     <PackageReference Include="CTA.Rules.PortCore" Version="1.8.33-alpha-g41d542cce7" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="NuGet.Protocol" Version="5.10.0" />
+    <PackageReference Include="NuGet.Protocol" Version="5.11.0" />

```

Comment:
Since this is in multiple projects, can we remove from parent projects and keep in referenced projects only? This comment applies to all packages referenced in more than one project.

Reply:
Implemented per our conversation- thanks!